agent/envoy_bootstrap/envoy_bootstrap.go

```diff
@@ -180,7 +181,22 @@ func getRuntimeConfigLayer0() (map[string]interface{}, error) {
 		// This also clears all the outlier detection counters. To enable the new behavior, set Envoy env variable
 		// ENVOY_ACTIVE_HEALTH_CHECK_UNEJECT_HOST to `true`.
 		"envoy.reloadable_features.successful_active_health_check_uneject_host": setActiveHealthCheckUnejectHost,
-	}, nil
+	}
+
+	// ====== Runtime config with no defaults set ======
+	// Not set by Default
+	// http: Add runtime flag http.max_requests_per_io_cycle for setting the limit on the number of HTTP requests processed
+	// from a single connection in a single I/O cycle. Requests over this limit are processed in subsequent I/O cycles.
+	// This mitigates CPU starvation by connections that simultaneously send high number of requests by allowing requests
+	// from other connections to make progress. This runtime value can be set to 1 in the presence of abusive HTTP/2 or HTTP/3
+	// connections. By default this limit is disabled.
+	if maxRequestsPerIoCycle, err := env.OrInt("MAX_REQUESTS_PER_IO_CYCLE", -1); err != nil {
```

Comment:
Is. -1 the default value this takes? Should it be 1 instead

Reply:
in line 195 it will make sure we don't sent default if it is less than 0